server/controllers/availability.js

```diff
@@ -7,13 +7,20 @@ const mongoose = require('mongoose')
 
   exports.updateAvailability = asyncHandler(async (req, res) => {
 
-     for(let prop in req.body.values.availability) if(!req.body.values.availability[prop]) delete req.body.values.availability[prop];//it will remove fields who are undefined or null
+     for(let prop in req.body.values.availability){
+      if(!req.body.values.availability[prop]){
+        delete req.body.values.availability[prop];
+        //it will remove fields who are undefined or null
+      }
+     }
 
-     availability = await Availability.findOneAndUpdate({_id: mongoose.Types.ObjectId(req.body.values._id)}, {availability: req.body.values.availability}, {new: true, strict: false});
+     availability = await Availability.findOneAndUpdate(
+    {_id: mongoose.Types.ObjectId(req.body.values._id)},
+    {availability: req.body.values.availability},
+    {new: true, strict: false});
 
      //if there is no availability for whatever reason- create it
-     if(!availability) {  
-      console.log('making new availability')
+     if(!availability) {
```

Comment:
`if (!availability) { . . . }`

Reply:
Please make sure we are writing clean code formatting, this is decreasing the codebase health and this is not what we want. Things like this are rather simple and don't need to show up in a review process. Thanks!

Reply:
OK